src/seedu/addressbook/Main.java

```diff
@@ -31,24 +32,31 @@
     private List<? extends ReadOnlyPerson> lastShownList = Collections.emptyList();
 
 
-    public static void main(String... launchArgs) {
+    public static void main(String... launchArgs) throws FileNotFoundException {
```

Comment:
As we discussed in the tutorial, we should handle the exception at the right place. In this example, the run() already handle the new exception, so there is no way for the main() to throws exception. :-)